pagopa/io-web-profile
# chore: [IOPID-1039, IOPID-1226] add FAQ

## Short description

Add the new FAQ and made some copy fixes.

## List of changes proposed in this pull request

- Fix Link import and import all Link from MUI/Italia and add style
- Add FAQ
- Fix text errors
- Componentize lists
- Export common logics

## How to test

Run the application and check that in the lock, unlock and logout cases there is the correct FAQ, that all links are figma-compliant.
